### PR TITLE
Prevent "Unexpected state: processing" error

### PR DESCRIPTION
**Pull request by marianvlad**
When I'm trying to upload a file, I'm getting a random "**Unexpected state: processing**" error to almost every file. This should fix the problem https://github.com/mcrapet/plowshare-modules-legacy/issues/115. No more duplicate files 😄

### CODE REVIEW

File: rapidgator.sh [hunk 1]
```diff
@@ -661,6 +661,9 @@ rapidgator_upload() {
             -F "file=@$FILE;type=application/octet-stream;filename=$DEST_FILE" \
             "$UP_URL$SESSION_ID&folder_id=$FOLDER_ID") || return
 
+        sleep 10
```
Comment (mcrapet) [inline rapidgator.sh]: don't use `sleep` but `wait`, this is the proper plowshare API to use.
10s for synchronising is huge, could you test with 8s?

**Comment by marianvlad** [comment]
Done

**Comment by mcrapet** [comment]
Applied thanks.

**Comment by mcrapet** [comment]
Some documentaion:
https://github.com/mcrapet/plowshare/wiki/Modules

**Pull request merged**